torchtune/training/_distributed.py

```diff
@@ -26,7 +26,7 @@
     StateDictOptions,
 )
 from torch.distributed.device_mesh import DeviceMesh, init_device_mesh
-from torch.distributed.fsdp import ShardingStrategy
+from torch.distributed.fsdp import FSDPModule, ShardingStrategy
```

Comment:
I think we don't need FSDPModule now, right?

Reply:
Eh, it's technically used as a signature in some of our utils - I'd rather just keep it in.